atomic_reactor/plugins/post_import_image.py

```diff
@@ -56,6 +58,12 @@ def __init__(self, tasker, workflow, imagestream, docker_image_repo,
         self.insecure_registry = insecure_registry
         self.retry_delay = retry_delay
 
+        if import_attempts < 1:
```

Comment:
@csomh could add a test with retry < 1, and ensure that it raises an exception and that exception msg includes "ValueError" and 'should be at least 1'. Not great, but will get the job done.